R/internals.R

```diff
@@ -1,3 +1,29 @@
+.rename_fb_cols <- function(df) {
```

Comment:
re-factored this out from `clean_advanced_stat_table()` so that i could use the renaming logic

Reply:
Nice